packages/@aws-cdk/aws-codedeploy/README.md

```diff
@@ -47,6 +47,49 @@ const deploymentGroup = codedeploy.ServerDeploymentGroupRef.import(this, 'Existi
 });
 ```
 
+#### Load balancers
+
+You can specify a load balancer with the `loadBalancer` property when creating a Deployment Group.
```

Comment:
Missing:
- Why would I want to do that? (you can copy & paste from the AWS docs)
- Reference to relevant AWS documentation?

Reply:
Done.